deploy/dev/kafka.yaml

```diff
@@ -45,6 +45,7 @@ spec:
             nodePort: 31092
           brokers:
           - broker: 0
+#            advertisedHost: host.docker.internal # swap these lines to enable Zeppelin, TODO: figure out a way around this
```

Comment:
We could theoretically have multiple listeners here, one for the `host.docker.internal` use-case, and one for the `localhost` use-case. For example, maybe `external` and `docker` could be separate "listeners" on different ports.

But the point is kinda moot of we just move Zeppelin inside K8s. All this stuff gets easier.

Reply:
Yea I tried this and it wouldn't work... didn't want to kill too much time on it